db/data/20220721145016_populate_permissions.rb

```diff
@@ -10,7 +10,8 @@ def up
       { name: 'ManageSiteSettings' },
       { name: 'ManageRoles' },
       { name: 'SharedList' },
-      { name: 'CanRecord' }
+      { name: 'CanRecord' },
+      { name: 'roomLimit' }
```

Comment:
Since we've pushed Alpha, we need to create new migrations - not push to old ones

Reply:
Remove this :)